presto-docs/src/main/sphinx/release/release-342.md

```diff
@@ -0,0 +1,49 @@
+# Release 342 (XX Sep 2020)
+
+## General Changes
+
+* Improve performance of queries that use decimal type. ({issue}`4886`)
+* Improve query performance by adding support for dynamic filtering and dynamic
```

Comment:
Maybe w should describe it in terms of SQL syntax, not internal operator. 
Sth like:
```
Improve performance of queries which use ``expression IN SELECT ...`` filters. This is achieved via additional support for dynamic filtering an dynamic partition pruning.
```